models/sipnet/R/write.configs.SIPNET.R

```diff
@@ -559,14 +567,27 @@ write.config.SIPNET <- function(defaults, trait.values, settings, run.id, inputs
         param[which(param[, 1] == "laiInit"), 2] <- lai
       }
 
-      #Initial LAI is set as 0 for deciduous forests and grasslands for non-growing seasons
-      if (!(lubridate::month(settings$run$start.date) %in% seq(5,9))){ #Growing seasons are coarsely defined as months from May to September for non-conifers in the US
-         site_pft <- utils::read.csv(settings$run$inputs$pft.site$path)
-         site.pft.name <- site_pft$pft[site_pft$site == settings$run$site$id]
-         if (site.pft.name!="boreal.coniferous") {   #Currently only excluding boreal conifers. Other evergreen PFTs could be added here later.
-              param[which(param[, 1] == "laiInit"), 2] <- 0       
-          }
+      # Sipnet always starts from initial LAI whether day 0 is in or out of the
+      # growing season -> set LAI=0 when a deciduous PFT starts with leaves off
+      #
+      # Note: At this writing in Jan 2025, leafOnDay and LeafOffDay are taken
+      # from the model defaults (template.param) unless:
+      # - settings$run$inputs$leaf_phenology is provided, or
+      # - the PFT sets leafOnDay/leafOffday as traits.
+      # So unless you set something different, it's probably using DOY 144/285
+      # ==> leaves are on from late May through mid-October.
+      is_deciduous_pft <- isTRUE(param[param[, 1] == "fracLeafFall", 2] < 0.5)
```

Comment:
I think the logic here is reverse. If a PFT is deciduous then leaf fall should be 1, while non-deciduous species should be near zero.

More complicated would be if someone tried to capture evergreen phenology with an intermediate value (e.g., 0.3 leaf loss would be reasonable for many confiers) you'd still need to invoke phenology but you wouldn't want to start from LAI = 0. LAI*fracLeafFall is an option, as noted in a comment below, but not only does that assume the LAI IC is near peak, but also that the LAI isn't already being nudged by the SDA (which, presumably, would see the seasonal cycling of LAI)

Reply:
Fixed in 8b6f55b